packages/ai-jsx/src/batteries/constrained-output.tsx

```diff
@@ -327,7 +330,26 @@ export async function* JsonChatCompletionFunctionCall(
 
   const frames = render(childrenWithCompletion);
   for await (const frame of frames) {
-    yield JSON.stringify(JSON.parse(frame).arguments);
+    const object = JSON.parse(frame).arguments;
```

Comment:
Let's call this `jsonResult` – `object` is super generic. Also, `json` doesn't have to be an object. :smile: